py3status/modules/rss_aggregator.py

```diff
@@ -100,10 +119,61 @@ def _get_count_owncloud(self):
         except:
             return None
 
+    def _get_count_ttrss(self):
+        try:
+            rss_count = 0
+            api_url = "%s/api/" % self.server
+            r = requests.post(api_url, data=json.dumps({
+                'op': "login",
+                'user': self.user,
+                'password': self.password
+            }))
+            sid = r.json()['content']['session_id']
+            if not self.feed_ids and not self.folder_ids:
+                r = requests.post(api_url, data=json.dumps({
+                    'sid': sid,
+                    'op': "getUnread"
+                }))
+                rss_count = r.json()['content']['unread']
+            else:
+                for folder in self.folder_ids:
+                    r = requests.post(api_url, data=json.dumps({
+                        'sid': sid,
+                        'op': "getFeeds",
+                        'cat_id': folder,
+                        'include_nested': True
+                    }))
+                    for item in r.json()['content']:
+                        rss_count += item['unread']
+                if self.feed_ids:
+                    r = requests.post(api_url, data=json.dumps({
+                        'sid': sid,
+                        'op': "getFeeds",
+                        'cat_id': -3
+                    }))
+                    for feed in r.json()['content']:
+                        if feed['id'] in self.feed_ids:
+                            rss_count += feed['unread']
+            requests.post(api_url, data=json.dumps({
+                'sid': sid,
+                'op': "logOut"
+            }))
+
+            return rss_count
+
+        except:
```

Comment:
I hate these raw excepts (I know why you've done it).

I think I need to make report_exception available to modules for this type of case so that we can log issues nicely.
anyhow I'm ok with it here.